tools/configure_test_cluster.py

```diff
@@ -1,170 +0,0 @@
-#!/usr/bin/env python3
-
-import argparse
-import logging
-import os
-import os.path
-import shutil
-import subprocess
-import sys
-import tempfile
-
-import cli_install
-import dcos_login
-
-logger = logging.getLogger(__name__)
-if __name__ == "__main__":
-    logging.basicConfig(level=logging.INFO, format="%(message)s")
-
-def _tools_dir():
-    return os.path.dirname(os.path.realpath(__file__))
-
-def _run_script(scriptname, args = [], **kwargs):
-    logger.info('Command: {} {}'.format(scriptname, ' '.join(args)))
-    script_path = os.path.join(_tools_dir(), scriptname)
-    # redirect stdout to stderr:
-    subprocess.check_call(['bash', script_path] + args, stdout=sys.stderr,
-                          **kwargs)
-
-
-class ClusterInitializer(object):
-    def __init__(self, aws_stack_id, auth_token, dns_address,
-            is_enterprise=False, security=None):
-        self.aws_stack_id = aws_stack_id
-        self.auth_token = auth_token
-        self.dns_address = dns_address
-        self.is_enterprise = is_enterprise
-        self.security = security
-        if not security:
-            self.dcos_url = 'http://%s' % dns_address
-        else:
-            self.dcos_url = 'https://%s' % dns_address
-        self._install_cli()
-
-
-    def create_mount_volumes(self):
-        fmt = 'Enabling mount volumes for cluster {} (stack id {})'
-        logger.info(fmt.format(self.dns_address, self.aws_stack_id))
-        import enable_mount_volumes
```

Comment:
@gabrielhartmann were you referring to this? 

This just calls main of this script: https://github.com/mesosphere/dcos-commons/blob/master/tools/enable_mount_volumes.py

And I did most definitely not remove that script in this change.